src/components/05_pages/Permissions/Permissions.jsx

```diff
@@ -1,12 +1,63 @@
-import React from 'react';
+import React, { Fragment, Component } from 'react';
 import { css } from 'emotion';
 
-const Permissions = () => (
-  <div>
-    <h1 className={styles.title}>Permissions</h1>
-    <p>This will be the permissions page.</p>
-  </div>
-);
+const drupalOrigin = 'http://localhost:9998';
+
+const Permissions = class Permissions extends Component {
+  state = {
+    loaded: false,
+  };
+  componentDidMount() {
+    fetch(`${drupalOrigin}/permissions?_format=json`)
```

Comment:
I think we should group the permissions when we render. Otherwise live filtering permissions would be a bit more pain. Thoughts?